source/agora/common/Config.d

```diff
@@ -114,11 +114,6 @@ public struct NodeConfig
     /// and TODO: addresses should be different prefix (e.g. TN... for TestNet)
     public bool testing;
 
-    /// The Unix timestamp of the Genesis block. Blocks are created at a
-    /// regular interval starting from this time.
-    public time_t genesis_start_time =
```

Comment:
Why did you remove this?

Reply:
We should still have a target average for the block interval, which should be based on the genesis start time.

Reply:
Ah you moved it to the genesis block header. That's alright then, but it would be good to make a comment in the commit message.